integration_tests/src/base_node_process.rs

```diff
@@ -185,7 +185,7 @@ pub async fn spawn_base_node_with_config(
             base_node_config.base_node.storage.pruning_interval = 1;
         };
 
-        base_node_config.base_node.grpc_server_deny_methods = vec![];
+        base_node_config.base_node.grpc_server_allow_methods = vec![];
 
```

Comment:
I suspect this list must be fully populated now instead of being left empty. 
Cucumber errors:
```
Captured output: called `Result::unwrap()` on an `Err` value: Status { code: PermissionDenied, message: "`ListConnectedPeers` method not made available"
```
```
Step panicked. Captured output: called `Result::unwrap()` on an `Err` value: ExitError { exit_code: GrpcError, details: Some("Base node not responding to gRPC requests: 'get_new_block_template'") }
```
Step panicked. Captured output: called `Result::unwrap()` on an `Err` value: Status { code: PermissionDenied, message: "`GetTipInfo` method not made available",
```